common/lib/xmodule/xmodule/capa_base.py

```diff
@@ -659,9 +663,9 @@ def get_problem_html(self, encapsulate=True):
         else:
             check_button = False
             check_button_checking = False
-
+        
         content = {
-            'name': self.display_name_with_default_escaped,
+            'name': self.display_name,
```

Comment:
You verified that the HTML file(s) using this content has the escaping filter set?